tests/test_numseq.py

```diff
@@ -45,10 +33,45 @@ def test_geo(self):
         """varyifying geo functions from range 0 to 9"""
         from numseq.geo import square, triangle, cube
 
+        saved_stdout = sys.stdout
+
+        out = StringIO.StringIO()
+        sys.stdout = out
+
         print("Geometric numbers (square, triangle, cube)")
         for i in range(10):
             print ("{}: {} {} {}".format(i, square(i), triangle(i), cube(i)))
 
+        output = out.getvalue().strip()
+
+        expected = open("./Geometric", "r").read()
+
+        self.assertEquals(output, expected)
+        sys.stdout = saved_stdout
+
+    def test_prime(self):
+        """varifying prime functions in prime module"""
```

Comment:
The summary line of docstrings should be capitalized and end with a period.
https://www.python.org/dev/peps/pep-0257/#one-line-docstrings